google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/cloudpubsub/internal/AckSetTracker.java

```diff
@@ -24,4 +24,8 @@
   // Track the given message. Returns a Runnable to ack this message if the message is a valid one
   // to add to the ack set. Must be called with strictly increasing offset messages.
   Runnable track(SequencedMessage message) throws CheckedApiException;
+
+  // Discard all outstanding acks and wait for any pending commit offset to be acknowledged by the
```

Comment:
Instead, just make this extend Flushable like the internal publisher? https://docs.oracle.com/javase/8/docs/api/index.html?java/io/Flushable.html

Reply:
Actually ignore this.

Reply:
Ack. I was going to say that I need to throw CheckedApiException.